packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -84,7 +89,13 @@ export const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
   );
 };
 
-export default connect(null, {
+const mapState = ({ pipeline }: RootState) => ({
+  isPipelineInvalid: pipeline.some((x) => !x.isValid),
```

Comment:
Should probably take into account if stage is disabled or not (so only check on stages that we will actually try to run)